File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/documenti/Documento.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AgronicaCoreModelsSTD.documenti
{
    public class Documento
    {
        public int ID_Tipologia { get; set; }
        public DateTime Data_Scadenza { get; set; }
        public String Descrizione { get; set; }
        public String Note { get; set; }
        public EnteRilascio Ente_Rilascio { get; set; }
        public DateTime Data_Rilascio { get; set; }
        public string Numero { get; set; }
        public List<DocumentoAllegato> Allegati { get; set; }

    }


}

File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/documenti/Documento.ts
import { DocumentoAllegato } from "./DocumentoAllegato";
import { EnteRilascio } from "./EnteRilascio";

export class Documento{
    ID_Tipologia: number;
    Data_Scadenza: Date;
    Descrizione: string;
    Note: string;
    Ente_Rilascio: EnteRilascio;
    Data_Rilascio: Date;
    Numero: string;
    Allegati: DocumentoAllegato[];
}
